BasicTests/crypto.json

```diff
@@ -0,0 +1,16 @@
+{
+    "aes1": {
```

Comment:
I have no idea how many teams are still using this files. perhaps it is better to leave it here as it is small